mitmproxy/flowfilter.py

```diff
@@ -15,7 +15,9 @@
                         application/javascript
                         text/css
                         image/*
-                        application/x-shockwave-flash
+                        application/x-shockwave-flash ?
```

Comment:
Should flash be included in the patterns too? It currently isn't.

Reply:
Thanks for catching this leftover. Flash nowadays would be a rather interesting find than anything else, we removed it from the asset list a while ago.